.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Bug report
+about: Unexpected behavior? Tell us more
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+# Describe your issue
+
+(Tell us what the problem is in a few words)  
+*Ex: As an end user, when I access the console and see a service inventory, if attributes have too long values, I can't see any of the right buttons anymore*
+
+# Steps to reproduce
+
+1. step 1
```

Comment:
I do not really understand why examples are given in the other sections but here you just have step1, step2, ... which is obvious from the numbers of course.

I do understand that it depends on the project that it depends. However, the current examples are all based on the same issue. So why not:

1. Go to the console
2. Go to the service catalog by clicking service catalog in the menu
3. Go to the service inventory of a service
4. When a service instance exists with a very long attribute the buttons on the right are no longer visible.